frontends/verilog/verilog_parser.y

```diff
@@ -169,6 +169,16 @@ static bool isInLocalScope(const std::string *name)
 	return (user_types.count(*name) > 0);
 }
 
+static AstNode *getTypeFromPackage(const std::string &pkg_name, const std::string &type_name)
+{
+	auto *pkg = ast_stack[0]->find_child(AST_PACKAGE, pkg_name);
+	if(pkg)
+	{
+		return pkg->find_child(type_name);
+	}
+	return nullptr;
+}
+
```

Comment:
Remove, this change is not necessary for multiranges support.

Reply:
I'm using this function to look for specific variable type here: https://github.com/antmicro/yosys/pull/532/files#diff-cfa391ea6199382f4c13a4d3f892e3eaa76ff9b24793ee31c045793afe71a550R1999